ouroboros-consensus-diffusion/CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
-# CHANGELOG
+# Changelog for ouroboros-consensus-diffusion
 
-This package is part of a bundle of packages. Please consult `CHANGELOG.md` in
-the package `ouroboros-consensus` at the same version as this package to see the
-current changelog that applies also to this package.
+# Changelog entries
+
+<a id='changelog-0.5.0.0'></a>
+## 0.5.0.0 - 2023-04-14
+
+### Breaking
+
+- Peer Sharing Integration:
+  - Monomorphized `Handlers` client field types to `ConnectionId addr`;
+    and added PeerSharing handlers.
+  - Changed `mkHandlers` function to receive a function to compute peer sharing addresses;
+  - Changed `Codecs` type and propagated changes to relevant functions (e.g. `defaultCodecs`);
```

Comment:
@bolt12 haven't we renamed `defaultCodecs` to `serialiseCodecs`?

Reply:
No that PR hasn't been merged yet, I forgot to address that bit

Reply:
Ignore this comment, your `defaultCodecs` is something different.